types/react-pivottable/index.d.ts

```diff
@@ -0,0 +1,11 @@
+// Type definitions for react-pivottable 0.11
+// Project: https://github.com/medipass/react-payment-inputs
```

Comment:
what is the module project page?

Reply:
ohhh... my god!!
https://www.npmjs.com/package/react-pivottable
That was so careless.